<launch>
    <env name="ROSCONSOLE_FORMAT" value="[${severity}][${thread}][${node}/${function}:${line}]: ${message}"/>
    <arg name="waypoints_topic" default="my_t3_waypoints"/>

    <node pkg='final_project' type='statemachine.py' name="custom_follow_waypoints"
    output="screen" clear_params="true">
        <param name="goal_frame_id" value="map"/>
        <param name="custom_waypoints_topic" value="$(arg waypoints_topic)"/>
    </node>
    <node pkg='final_project' type='waypoint.py' name="moving_to_waypoints"/>

</launch>